Conversions/FormatDuration.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Formats a duration in milliseconds into a human-readable string.
+ *
+ * @param {number} ms - The duration in milliseconds (can be negative).
+ * @returns {string} A formatted string representing the duration.
+ */
+const FormatDuration = (ms) => {
+  if (ms < 0) ms = -ms
```

Comment:
Why is this useful or reasonable? I'd remove it altogether and just not allow negative durations.